app/src/main/java/cm/aptoide/pt/abtesting/ABTestActionBody.java

```diff
@@ -0,0 +1,41 @@
+package cm.aptoide.pt.abtesting;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by franciscocalado on 19/06/18.
+ */
+
+public class ABTestActionBody {
```

Comment:
Isn't this and ABTestImpressionBody the same structure? Could we merge this someway and just inject or set the differences (which it seems like it is only the assignment name)?

Reply:
Fixed